### DISCUSSION TITLE
Keyboard not working on remote Mac

### DISCUSSION THREAD

**Discussion by bansonln** [rustdesk/rustdesk#976] [General]
I am connecting from a Mac (Monterey 12.4) to Mac (Catalina 10.15.7) and I can use the mouse but the keyboard will not work. I can't type anything. I have rebooted the target machine but I still can't get it to work. Any help would be greatly appreciated.

**Comment by bansonln** [comment]
I did try using the web client and it worked but it is very slow. I am trying to use the app. Version 1.1.9

**Comment by rustdesk** [comment]
Yes, it is a new issue arising from latest MacOS update.

https://github.com/rustdesk/rustdesk/issues/974#issuecomment-1185644923

There is a stupid work around mentioned in above link, you can try out.

@abhizer is looking for a fix.
